CHANGELOG.rst

```diff
@@ -1,5 +1,62 @@
 .. _changelog:
 
+0.6.3
+-----
+
+New feature
+~~~~~~~~~~~~
+
+⭐️ Project: commit list is now displayed
+
+Notable improvements
+~~~~~~~~~~~~~~~~~~~~
+
+🚄 Datasets: dataset creation and import unified in the UI
+
+🚄 Environments: auto-saved branches are filtered per username
+
+🚄 Improve markdown rendering and highlight code
+
+🚄 Editing markdown files is easier as ``ckeditor`` is partially integrated inside Renku
+
+Bug fixes
+~~~~~~~~~
+
+* Dataset contains all folders from unzipped file
+* If a dataset fails, non-failed datasets are displayed
+* Improved UX for when datasets take too long
+* Datasets: no failure when adding ignored files
+
+Miscellaneous
```

Comment:
Renku Template Updates instead of "Miscellaneous"?

Reply:
usually release notes are not organized by components